packages/itwinui-react/package.json

```diff
@@ -125,15 +125,15 @@
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-require-extensions": "^0.1.3",
     "jsdom": "^24.0.0",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
     "typescript": "5",
     "vite": "~5.4.14",
     "vitest": "^1.6.1"
   },
   "peerDependencies": {
-    "react": ">=17.0.0 <19.0.0",
-    "react-dom": ">=17.0.0 <19.0.0"
+    "react": ">=17.0.0 <20.0.0",
+    "react-dom": ">=17.0.0 <20.0.0"
```

Comment:
Can we drop the `<20.0.0` and just allow anything above `>=17.0.0`? I see no benefit in being restrictive, just in case it already works when React v20 is released.